pkg/reconciler/apiserversource/resources/config.go

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
it would be nice to move this down to `pkg` as we are going to use it in many places.
@n3wscott what do you think?

Reply:
How about `pkg/utils`?